dashboard/test_dashboard.py

```diff
@@ -4,4 +4,4 @@
 
 
 def test_placeholder():
-    assert path.exists('README.md')
+    assert False
```

Comment:
Should this be failing?

Reply:
Yes, I wanted to see if the overall workflow passes, given by branch name doesn't start with `dashboard-`.

Reply:
Very well.

Reply:
Can do

Reply:
otherwise the workflow wouldn't break, but it would cause a failing test in another part of the project